storage.rb

```diff
@@ -0,0 +1,67 @@
+require 'json'
+
+module Storage
+  def load_albums
+    json_albums = File.read('./local/music_album.json')
+    albums_data = JSON.parse(json_albums)
+    albums_data.each do |element|
+      album = MusicAlbum.new(element['name'], element['publish_date'], element['on_spotify'])
+      full_item = [album, element['genre']]
+      @music_albums << album
+      @full_items << full_item
```

Comment:
Here, @rpire. We can push the objects directly to @music_albums and @full_items to keep it simple and dry.